pkg/sql/randgen/mutator.go

```diff
@@ -742,10 +742,21 @@ func postgresCreateTableMutator(
 			for _, def := range stmt.Defs {
 				switch def := def.(type) {
 				case *tree.ColumnTableDef:
-					colTypes[string(def.Name)] = tree.MustBeStaticallyKnownType(def.Type)
+					colType := tree.MustBeStaticallyKnownType(def.Type)
+					// Disable locales because CRDB random generator generates
+					// locales without double-quotes, and they are unknown to
+					// Postgres (e.g. `i TEXT COLLATE de_DE` won't work with
+					// Postgres).
+					if colType.Family() == types.CollatedStringFamily {
+						colType.InternalType.Family = types.StringFamily
```

Comment:
i think this would be simpler if it was:
```
if colType.Family() == types.CollatedStringFamily {
  colType = types.String
}
```

Reply:
done